cg/meta/archive/ddn.py

```diff
@@ -0,0 +1,140 @@
+"""Module for archiving and retrieving folders via DDN."""
```

Comment:
```suggestion
"""Module for archiving and retrieving folders via DDN Dataflow."""
```
DDN is the  company and Dataflow is the software. I think we should rename the module and API to DataFlow instead of DDN.